profile/README.md

```diff
@@ -1 +1,8 @@
-Welcome!
+## Welcome to the OpenVINO™ GitHub organization!
+[OpenVINO™](https://en.wikipedia.org/wiki/OpenVINO) is an open-source toolkit for optimizing and deploying AI inference.
+* [openvino](https://github.com/openvinotoolkit/openvino) is the main repository with the sourse code for runtime and some tools.
+* [nncf](https://github.com/openvinotoolkit/nncf) Neural Network Compression Framework is used for enhanced OpenVINO™ inference to get performance boost with minimal accuracy drop.
```

Comment:
Is it correct to say NNCF is "used for enhanced OpenVINO™ inference"? I'm worried it could be misread. Maybe something like "used for post-training and quantization-aware training optimization, to speed up OpenVINO inference" ... Or "used to optimize models for even better performance with OpenVINO."

Reply:
This wording is already used for NNCF repo description. 
See about section: https://github.com/openvinotoolkit/nncf

Reply:
That's true... If we can confirm if this statement is correct or not, we can make sure the proper wording is used in both descriptions, as well as the docs pages.